.node-version

```diff
@@ -1 +1 @@
-18.13.0
+20.9.0
```

Comment:
Don’t set node v20 since it’s still in beta on Vercel ( https://vercel.com/changelog/nodejs-20 )

Reply:
fixed!

Reply:
@KoukiMinamoto I don't know why you raised the version, but are there any negative effects of reverting the node version(20 -> 18)? have you tested it?  anyway, I will approve this PR if everything is fine!

Reply:
There is no nagative effects! I raised it cus my pnpm was working something strange on 18.13.0. but it's resolved already!

Reply:
great - LGTM